app/assets/stylesheets/action_page.scss

```diff
@@ -14,64 +31,48 @@
 
 #description {
   margin-top: 3em;
-  h3 {
-    color: #0a639b;
-    line-height: 1.3;
-    * { background-color: #fff; }
-  }
   p, li {
     margin-bottom: 1em;
   }
 }
 
-.tools-wrapper {
-  z-index: 10;
-}
-
 .form-control {
   border-radius: 0;
   padding: 0 0.5em;
   font-size: 0.8em;
 }
 
-#share-buttons-wrapper {
-  position: absolute;
-  text-align: center;
-  right: 20px;
-  top: 0px;
-  display: none;
-  @media screen and (min-width: $md) {
-    position: relative;
-    right: 0;
-    #share-buttons a {
-      display: inline-block !important;
-    }
-  }
-}
-
 #tools {
-
   .tool {
-    background-color: #ffffff;
+    background-color: $gray-darker;
+    color: black;
     margin-bottom: 20px;
-    border: solid 1px #cc0000;
-    -webkit-box-shadow: 0 0 5px 5px rgba(0, 0, 0, 0.1); // iOS <4.3 & Android <4.1
-    box-shadow: 0 0 2px 2px rgba(0, 0, 0, 0.1);
 
+    a {
+      color: black;
+      border-bottom: 2px solid black;
+      &:hover {
+        border-bottom: 2px solid $charcoal;
+      }
+    }
     .tool-container {
-      padding: 1em 0.4em 0 0.4em;
+      padding: 2em 0.4em 0 0.4em;
     }
 
     .tool-heading {
-      color: #cc0000;
+      color: white;
       text-transform: uppercase;
       text-align: center;
 
       .tool-title {
-        text-transform: none;
-        font-size: 1.2em;
+        color: white;
+        font-family: $league;
+        font-size: 4rem;
```

Comment:
The mockup seems to anticipate a short tool title like this:

![Screenshot_2019-06-26_15-07-21](https://user-images.githubusercontent.com/6469642/60218844-652a6580-9825-11e9-9726-a7ee758929ab.png)

but we allow those titles to be customized, so we have [some actions](https://act.eff.org/action/california-demand-community-control-of-police-surveillance-2b7ce8a3-0f59-4164-8d6a-1adb8fc306f4) where instead it'll look like

![Screenshot_2019-06-26_14-52-49](https://user-images.githubusercontent.com/6469642/60218169-60fd4880-9823-11e9-94b6-6673ccf65c1d.png)

Something closer to 2rem / not uppercased might be better?

Reply:
Asking design folks.

Reply:
It appears the Activism might be okay with this always saying "Take Action". So I'm going to punt that issue for now.